eCommerce/product/management/commands/seed_db.py

```
@@ -0,0 +1,107 @@
+                name=template["name"],
+                category=random.choice([c for c in category_objs if c.name == template["category"]]),
+                brand=product_brand,
+                description=f"A high-quality {template['name']} from {product_brand.name}.",
+            )
+
+            product_colour = random.choice(colour_objs)
+            product_item = ProductItem.objects.create(
+                product=product,
+                colour=product_colour,
+                sku_base=f"{product.brand.name.upper()[:3]}-{product.name.replace(' ', '').upper()[:5]}-{product_colour.colour_name.upper()[:3]}",
+                original_price=round(random.uniform(20.0, 100.0), 2),
```

Ruff (0.14.6)

Standard pseudo-random generators are not suitable for cryptographic purposes

(S311)